chaos_kitten/toys_cli.py

```
@@ -0,0 +1,225 @@
+    toys_dir = _get_local_toys_dir()
+    # Determine extension from URL or default to yaml
+    ext = ".json" if download_url.endswith(".json") else ".yaml"
+    dest_path = toys_dir / f"{slug}{ext}"
+    
+    if dest_path.exists() and not force:
+        console.print(f"[yellow]Profile '{slug}' is already installed at {dest_path}. Use --force to overwrite.[/yellow]")
+        return
+        
+    console.print(f"[cyan]Downloading '{slug}' from {download_url}...[/cyan]")
+    try:
+        req = urllib.request.Request(download_url, headers={'User-Agent': 'Chaos-Kitten-CLI'})
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)